pkg/controller/kogitoapp/test/crd_validation_test.go

```diff
@@ -37,21 +37,6 @@ func TestSampleCustomResources(t *testing.T) {
 	}
 }
 
-func TestExampleCustomResources(t *testing.T) {
```

Comment:
Why is this test removed?

Reply:
The example resources now contains a lot of code/yaml files that are not always related to the CRDs. Also, for Operator SDK 0.17.1, we will need to move the `examples` directory from there. It's not worth keeping it any more. Maybe refactoring to something else in the future.